doc/GetShellWithoutMonitorOnLinux.md

```diff
@@ -31,6 +31,6 @@ These instructions will configure a Raspberry Pi so that you can proceed with th
     ```
     -- Files updated and ready for Wifi and SSH over USB --
     ```
-1. Eject the SD card safely, insert into your Pi, and reboot. If the Pi is connected over USB to your host, and/or if the Wifi setup went correctly, you should be able to `ssh pi@raspberrypi.local`. The default password is `raspberry`.
```

Comment:
This document is about doing a manual install of regular Raspbian (none of these steps are needed when using the prebuilt image), in which case the hostname is "raspberrypi" as shown here.